extensions/cornerstone/src/components/CinePlayer/CinePlayer.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect } from 'react';
+import { CinePlayer, useCine, useViewportGrid } from '@ohif/ui';
+import { Enums, eventTarget } from '@cornerstonejs/core';
+
+function WrappedCinePlayer({
+  enabledVPElement,
+  viewportIndex,
+  servicesManager,
+}) {
+  const { toolbarService } = servicesManager.services;
```

Comment:
Could you consider getting the customization service here and if the customization service defines a cine player, then returning that CINE player instead of this one?  That way the user can replace the CINE player entirely should the mode decide they want to do that.  I've seen several implementations of different CINE players, so it is clearly something that people want to customize in the mode.